.github/workflows/osrm-backend.yml

```diff
@@ -234,7 +234,7 @@ jobs:
             CCOMPILER: gcc-7
```

Comment:
How I have tested this: since we do not have Apple Silicon based CI we cannot run tests there, so I just downloaded binaries from CI artefacts and ran tests locally putting these binaries to ./lib/binding folder before it.